react-common/components/controls/EmbedVideo.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { classList, ControlProps } from "../util";
+
+export interface EmbedVideoProps extends ControlProps {
+  src: "youtube" | "streams";
+  id: string;
+  autoplay?: boolean;
+  allowFullScreen?: boolean;
+}
+
+
+
+export const EmbedVideo = (props: EmbedVideoProps) => {
+    const {
+       src,
+       id,
+       autoplay,
+       allowFullScreen
+    } = props;
+
+
+
+    let videoURL = `https://www.youtube.com/embed/${id}?${autoplay ? "autoplay=1" : ""}`;
+
+
+
+    let streamURL = `https://msit.microsoftstream.com/video/${id}`;
```

Comment:
I don't think we have legal rights to add msit streams video for external service.  Please remove this for now.

Reply:
Will do, thanks for feedback!